README.md

```diff
@@ -96,10 +96,10 @@ Any help is appreciated.
 
 Here are some ways to help us:
 
- * add missing documentation for classes, roles, methods or operators
- * add usage examples to existing documentation
- * proofread and correct the documentation
- * tell us about missing documentation by opening issues on github.
+ * Add missing documentation for classes, roles, methods or operators.
+ * Add usage examples to existing documentation.
+ * Proofread and correct the documentation.
+ * Tell us about missing documentation by opening issues on github.
```

Comment:
Maybe use a nicer spelling for GitHub and linking the issues from here?

Reply:
Change github →GitHub and also wrap "opening issues in GitHub" with a link to the issues https://github.com/perl6/doc/issues